examples/SAP-to-Database/types.bal

```diff
@@ -0,0 +1,34 @@
+import ballerina/constraint;
+
+type S4HANAClientConfig record {|
+    string hostname;
+    string username;
+    string password;
+|};
+
+type DatabaseConfig record {|
+    string host;
+    string username;
+    string password;
+    string databaseName;
+    int port;
+|};
+
+type InspectionSpecificationRec record {
```

Comment:
```suggestion
type InspectionSpecification record {
```

Reply:
Where is this record used?

Reply:
This record has attributes that should be stored in database. It's used to get the required attributes from an Inspection specification retrieved from master data to insert into database.  I have used an array of this record type to get the inspection specification details when they are checked with the database so it will only hold the necessary attributes only.